blakserv/roofile.c

```diff
@@ -956,6 +956,14 @@ bool BSPGetStepTowards(room_type* Room, V2* S, V2* E, V2* P, unsigned int* Flags
    if (!Room || !S || !E || !P || !Flags)
       return false;
 
+   bool moveOutsideBSP = False;
+
+   if ((*Flags & 0x100000) == 0x100000)
```

Comment:
Placeholder for a constant - we were discussing whether to fix the AI_MOVE_WALKTHROUGH_WALLS at all in IRC, I verified this approach works but wasn't sure it was the best one so it's uploaded in a mostly complete state.